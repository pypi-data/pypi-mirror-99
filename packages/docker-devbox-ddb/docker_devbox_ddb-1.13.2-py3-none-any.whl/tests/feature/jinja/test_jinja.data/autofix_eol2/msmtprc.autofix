account        mail
host           mail.{{jsonnet.docker.compose.network_name}}
port           25
from           postmaster@{{core.domain.sub}}.{{core.domain.ext}}
account default : mail
