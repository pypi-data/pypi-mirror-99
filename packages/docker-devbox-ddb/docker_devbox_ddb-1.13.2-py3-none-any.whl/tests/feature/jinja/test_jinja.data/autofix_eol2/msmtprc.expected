account        mail
host           mail.network_default
port           25
from           postmaster@domain.tld
account default : mail
